### PR TITLE
cheatsheets: add imagemagick cheatsheet

### PR DESCRIPTION
**Pull request by championswimmer**
screenshot from 2016-02-18 09 13 02

---
https://duck.co/ia/view/imagemagick_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@championswimmer Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@championswimmer Thanks for contributing and welcome to DuckDuckHack!

Please don't forget to add a description and example queries to your Instant Answer page :+1:

File: share/goodie/cheat_sheets/json/imagemagick.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "name": "ImageMagick",
+    "description": "ImageMagick includes a number of command-line utilities for manipulating images.",
+
+    "metadata": {
+        "sourceName": "ImageMagick",
+        "sourceUrl" : "http://www.imagemagick.org/script/command-line-tools.php"
+    },
+
+    "aliases": [
+        "imagemagick", "convert", "mogrify", "image manipulating"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/imagemagick.json]: `imagemagick` is already the default, so we don't need an alias for it. We probably should have `image magick` though.

Please remove `convert` and `image manipulating` -- "convert cheat sheet" would trigger this which isn't right. Likewise "image manipulation cheat sheet" shouldn't show this.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/imagemagick_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Imagemagick Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/imagemagick_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Imagemagick Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/imagemagick_cheat_sheet_mobile.png)

**Comment by championswimmer** [comment]
@moollaza I've made the required changes

**Comment by championswimmer** [comment]
I am not sure why the build failed, doesn't look like it was due to imagemagick cheatsheet. That built fine as per the build log

**Comment by championswimmer** [comment]
@moollaza done

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Imagemagick Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/imagemagick_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@championswimmer Thanks! This LGTM :+1:

**Comment by tagawa** [comment]
@championswimmer, thanks again for creating the ImageMagick cheat sheet! You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in command line tools.

When you're ready to move on, please:

- Create a new Instant Answer for searches you do related to command line tools

OR

- Test live command line Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by championswimmer** [comment]
I'll definitely love to help with points a and b

And also with more free API integrations.

**Comment by tagawa** [comment]
@championswimmer That's great - thank you for the quick reply. Yes, there are tons of free APIs out there that we could probably use to power more Instant Answers.

**Pull request merged**